### PR TITLE
Soften the code colors, and add light background

### PR DESCRIPTION
**Pull request by twalker**
### Summary

This is an attempt at applying a "Light Nature Theme" suggested in #82 
I am **not** a designer, but found complementary colors of the existing `blue` and `navy` colors, and tried to make some small but impactful changes to soften the pages with natural colors.

More specifically:
* Added a "lakeside light" css theme to the `code` renderings
* Added some margin around the `main` content.
* Added a light blue background.

As with Art, design is subjective. I will **not** be offended if either of you don't care for these design choices.

### What kind of change does this PR introduce?

- [x] Bug fix

Related issue: #82

### How to verify locally

1. `npm start`
1. http://localhost:8080/
 

**Expected results**:
Design should be less intimidating and more calming.
### Design Before
design-before

### Design After
designed-result

### CODE REVIEW

File: index.njk [hunk 1]
```diff
@@ -1,8 +1,5 @@
 ---
 layout: layouts/home.njk
-eleventyNavigation:
```
Comment (twalker) [inline index.njk:3]: Having "Home" in the navigation was duplicative of the link in the "Free Free Api Education" link. The Home link was also causing the navigation items to wrap to a second line at the narrower width. I removed Home from the navigation to address these 2 issues.

File: css/index.css [hunk 2]
```diff
@@ -19,7 +21,17 @@ body {
   margin: 0;
   font-family: -apple-system, system-ui, sans-serif;
   color: var(--darkgray);
-  background-color: var(--white);
+  background-color: var(--light-blue);
```
Comment (ghost) [inline css/index.css:24]: Looks great to me. I have a hard time choosing colors. @cmneal26 do you like this color choice?

**Review by ghost — APPROVED** [review]

File: css/index.css (hunk 2 above)
Comment (ghost) [reply css/index.css:24]: @twalker this isn't about your style choices. Just an honest question. Why no plant imagery?

Comment (ghost) [reply css/index.css:24]: Do you not like plants, or is this about you showing it as a portfolio item to potential technical writers? Or something else?

**Pull request merged**